<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDebugPanel</class>
 <widget class="QWidget" name="CDebugPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DebugPanel</string>
  </property>
  <widget class="QPushButton" name="btnInsertTraces">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert Traces</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEnumTraces">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Enum Traces</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="TracesToAddSpinBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SleepTimeBetweenInsertSpinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
   <property name="singleStep">
    <number>200</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
